### PR TITLE
[13.0][MIG] hr_timesheet_sheet_policy_project_manager

### PR DESCRIPTION
**Pull request by davejames**

### CODE REVIEW

File: hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py [hunk 1]
```diff
@@ -50,10 +47,9 @@ def _get_overlapping_sheet_domain(self):
     def _check_overlapping_sheets_project_id(self):
         self._check_overlapping_sheets()
 
-    @api.multi
     @api.constrains("company_id", "project_id")
     def _check_company_id_project_id(self):
-        for rec in self.sudo():
+        for rec in self.with_user(SUPERUSER_ID):
```
Comment (alexey-pelykh) [inline hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py]: @pedrobaeza see, not only I treat that instruction like that :)

Comment (alexey-pelykh) [reply hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py]: @davejames `sudo()` is not deprecated, `sudo(other_that_superuser)` is

**Review by alexey-pelykh — COMMENTED** [review]
Minor change needed, and please squash commits as per https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch

File: hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py (hunk 1 above)
Comment (pedrobaeza) [reply hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py]: Please propose an alternate redaction for that.

Comment (alexey-pelykh) [reply hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py]: Easy!

> Replace `sudo(user)`, keep `sudo()`: "deprecated use of sudo(user), use with_user(user) instead"

**Review by alexey-pelykh — APPROVED** [review]

File: hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py (hunk 1 above)
Comment (davejames) [reply hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py]: thanks good to know

Comment (pedrobaeza) [reply hr_timesheet_sheet_policy_project_manager/models/hr_timesheet_sheet.py]: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0/_compare/ac45f661a0aad55f05ed5b72f9e24d654022e18b...e548c9f190738f52d6ef0c5954556fba38205033

**Comment by davejames** [comment]
i think the codecov test is failing due to the hr_timesheet_sheet module, is there anything for me to do here?

**Comment by alexey-pelykh** [comment]
@davejames not required, yet you could add more tests to increase coverage

**Comment by davejames** [comment]
@astirpe @pedrobaeza if anyone could review this I would really appreciate it. I have another migration ready to submit but it depends on this module

**Comment by pedrobaeza** [comment]
Sorry, but I'm not user of this module.

**Review by astirpe — APPROVED** [review]
Code LGTM. Code review only, not tested.

**Pull request merged**